extensions/wikia/EmbeddableDiscussions/models/DiscussionsThreadModel.class.php

```diff
@@ -43,6 +43,16 @@ private function getUpvoteRequestUrl( $id ) {
 		return self::DISCUSSIONS_API_BASE_DEV . "$this->cityId/votes/post/$id";
 	}
 
+	private function getBaseUrl() {
+		global $wgDevelEnvironment;
+
+		if ( empty( $wgDevelEnvironment ) ) {
+			return self::DISCUSSIONS_API_BASE;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)